convert2rhel/systeminfo.py

```diff
@@ -359,6 +364,80 @@ def corresponds_to_rhel_eus_release(self):
         """
         return self.releasever in EUS_MINOR_VERSIONS
 
+    def _get_dbus_status(self):
+        """
+        Check whether dbus is running.
+
+        :returns: True if dbus is running.  Otherwise False
+        """
+        retries = 0
+        status = False
+
+        while retries < CHECK_DBUS_STATUS_RETRIES:
+            if self.version.major <= 6:
+                status = _get_dbus_status_from_sysv()
+            else:
+                status = _get_dbus_status_from_systemd()
```

Comment:
See review comment for `_get_dbus_status_from_systemd()`
```suggestion
                status = _check_dbus_service_running()
```